lib/plugin.js

```diff
@@ -63,10 +64,15 @@ ManifestPlugin.prototype.apply = function(compiler) {
 
     // module assets don't show up in assetsByChunkName.
     // we're getting them this way;
+    var allowCopied = this.opts.includeCopiedAssets;
```

Comment:
Remove the option, it will added as a breaking change, so no need to be backward compatible